run-smoke-tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+cd test/fixtures && python -m SimpleHTTPServer 9999 &
+
+./cli.js http://localhost:9999/online-only.html > results
+
+if ! grep -q "URL responds with a 200 when offline: false" results; then
```

Comment:
would much rather test non-pretty-printed results than strings, as the first is more likely to be stable over time than the second, and easier for humans to quickly scan and keep the test expectations up to date. We can do that in a follow up PR if you want, though.